requirements.txt

```diff
@@ -28,3 +28,4 @@ pylint>=2.2.2
 astroid>=2.1.0
 isort>=4.3.4
 setuptools>=20.7.0
+pre-commit>=2.2.0
```

Comment:
should we pin `cmake-format==0.6.9` just like we pinned `autopep8` and `pycodestyle`?